Remember The Date/View Controllers/ZenHelpViewController.m

```diff
@@ -95,6 +96,24 @@ -(void) setupSupportInformation {
     }];
 }
 
+- (void) setupSDK {
+    static dispatch_once_t onceToken;
```

Comment:
Yeah that was a bit careless. I made it static so that should persist for app life cycle.